MACM/MACMInitialization.py

```diff
@@ -352,42 +357,29 @@ def extractInfoIDProbDists(in_events, in_umapping):
         orderedInfoids[infoid_to_index[k]] = k
     
     dfprobs = pd.DataFrame(data=probs, index=orderedInfoids, columns=orderedInfoids)
-    dfprobs.to_csv("MACM_Init_Endogenous_ContentIDProbDists.csv",index_label='parentNarrative')
+    dfprobs.to_csv(f"{os.path.dirname(os.path.abspath(__file__))}/../init_data/MACM_Init_Endogenous_ContentIDProbDists.csv",index_label='parentNarrative')
```

Comment:
@deamonpog would be great to have the output folder as a parameter. Fine for now, but please fix in a new PR.